src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -13662,7 +13655,6 @@ void WriteCO2ResilienceTables(EnergyPlusData &state)
 
     // Using/Aliasing
     if (state.dataGlobal->NumOfZones > 0) {
-        int columnNum = 3;
         std::vector<int> columnHead = {state.dataOutRptPredefined->pdchCO2HourSafe,
```

Comment:
This can probably be a `std::array` also given that you've change the template variable from `std::array<Real64,columnNum>` to `columnNum`. `columnNum` can be used to parameterize both `std::array<Real64, columnNum>` and `std::array<int, columnNum>`.

Reply:
Yes, make sense, header should have the same size as number of columns, as well as columnMin, columnMax. I'll parameterize all of them with columnNum. I'll also use constexpr in place of the numbers like 5.